RecoPixelVertexing/PixelLowPtUtilities/plugins/ClusterShapeSeedComparitor.cc

```diff
@@ -89,7 +89,9 @@ PixelClusterShapeSeedComparitor::compatible(const SeedingHitSet  &hits,
 { 
     if (!filterAtHelixStage_) return true;
 
-    if(!helix.isValid()) edm::LogWarning("InvalidHelix") << "PixelClusterShapeSeedComparitor helix is not valid, result is bad";
+    if(!helix.isValid() //check still if it's a straight line, which are OK
+       && !helix.circle().isLine())//complain if it's not even a straight line
+      edm::LogWarning("InvalidHelix") << "PixelClusterShapeSeedComparitor helix is not valid, result is bad";
```

Comment:
@makortel @VinInn 
I wanted to check with you that I have correctly interpreted this case of using a "degraded" FastHelix.

Reply:
Interesting.... is the efficiency improving?

Reply:
Actually wonder what 
```float xc = helix.circle().x0(), yc = helix.circle().y0();```
would then do

Reply:
there is no change in physics in this PR, it just reduces the range of cases when the logWarning is issued.

The xc,yc are (0,0) for the case of a line.

Reply:
ok, we need to have a further look on how to use a straight line

Reply:
In the LogWarnings that I have investigated
this code is called by RecoTracker/TkSeedGenerator/plugins/SeedFromConsecutiveHitsTripletOnlyCreator.cc:53
there the code already apparently makes some assumptions about ```!helix.isValid()``` and already makes up a state at vertex for an invalid helix which simply points from the vertex to the outer hit (just as if it's a straight line)